db/compaction/compaction_picker_universal.cc

```diff
@@ -374,7 +383,7 @@ UniversalCompactionBuilder::CalculateSortedRuns(
 Compaction* UniversalCompactionBuilder::PickCompaction() {
   const int kLevel0 = 0;
   score_ = vstorage_->CompactionScore(kLevel0);
-  sorted_runs_ = CalculateSortedRuns(*vstorage_);
+  sorted_runs_ = CalculateSortedRuns(*vstorage_, earliest_mem_seqno_);
```

Comment:
As mentioned in the PR summary, we couldn't know the output level at this point therefore we might end up filtering out such files even when we are not compacting to L0 level and the filtered files are harmless.

Would this be problematic?? Maybe it's fine once we accumulate some data in memetable and increase the earliest_mem_seqno so that previously filtered out files can be chosen again.

Reply:
Actually I can look into whether multi-level universal compaction has something like IntraL0Compaction(). If there isn't such thing, then we can assume multi-level universal compaction always compact to Ln>0 and single-level always compact to L0. Downside is we are making this assumption that's subtle and might not get updated if we ever want to have IntraL0Compaction for multi-level universal.